db/seeds.rb

```diff
@@ -17,28 +17,28 @@ def generate_categories
 
   def generate_items
     @items = Item.create([
-                     { title: 'pizza',    description: "some cheese stuff", price: "5000" },
-                     { title: 'pizza2',   description: "some cheese stuff", price: "6000" },
-                     { title: 'pizza3',   description: "some cheese stuff", price: "7000" },
-                     { title: 'pizza4',   description: "some cheese stuff", price: "54000" },
-                     { title: 'pizza5',   description: "some cheese stuff", price: "50300" },
-                     { title: 'pizza6',   description: "some cheese stuff", price: "50" },
-                     { title: 'pizza7',   description: "some cheese stuff", price: "50030" },
-                     { title: 'pizza8',   description: "some cheese stuff", price: "500" },
-                     { title: 'pizza9',   description: "some cheese stuff", price: "5000" },
-                     { title: 'pizza10',  description: "some cheese stuff", price: "4000" },
-                     { title: 'pizza11',  description: "some cheese stuff", price: "1000" },
-                     { title: 'pizza12',  description: "some cheese stuff", price: "500" },
-                     { title: 'pizza13',  description: "some cheese stuff", price: "800" },
-                     { title: 'pizza14',  description: "some cheese stuff", price: "500" },
-                     { title: 'Veggie Pizza',  description: "Mushrooms, onions, peppers, and olives on a pizza", price: "300" },
-                     { title: 'veggies',  description: "an assortment of grilled veggies", price: "500" },
-                     { title: 'drink1',   description: "Coke",              price: "200" },
-                     { title: 'drink2',   description: "water",             price: "800" },
-                     { title: 'drink3',   description: "milk",              price: "500" },
-                     { title: 'drink4',   description: "orange juice",      price: "300" },
-                     { title: 'dessert1', description: "some cheese sweet stuff", price: "200" },
-                     { title: 'Caesar salad', description: "Just a regular Caesar salad", price: "800" }
+                     { title: 'pizza',    description: "some cheese stuff", unit_price: "5000" },
```

Comment:
Line is too long. [97/80]<br>Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.